packages/vite/src/node/server/moduleGraph.ts

```diff
@@ -35,6 +37,18 @@ export class ModuleNode {
   ssrError: Error | null = null
   lastHMRTimestamp = 0
   lastInvalidationTimestamp = 0
+  /**
+   * entryPoint of the module
+   *   - null - no init
+   *   - self - it's entry pointer (the ModuleNode what import by dynamic import)
```

Comment:
```suggestion
   *   - self - it's entry pointer (the ModuleNode which is imported by dynamic import)
```

is this better wording?